addons/website_project_issue/controllers/main.py

```diff
@@ -58,6 +64,23 @@ def my_issues(self, page=1, date_begin=None, date_end=None, sortby=None, filterb
         if date_begin and date_end:
             domain += [('create_date', '>', date_begin), ('create_date', '<=', date_end)]
 
+        # search
+        if search and search_in:
```

Comment:
Well removing search_all variable could lead to code easier to read. In expression.py there are some tool methods (combine, AND, OR) that would help you writing a bit more ast-like code (a bit ^^). This remark apply to similar code in other addons obviously.